node_modules/gh-events/lib/api.js

```diff
@@ -244,7 +248,18 @@ var updateEvent = module.exports.updateEvent = function(ctx, id, update, callbac
                     return callback({'code': 401, 'msg': 'You are not allowed to update this event'});
                 }
 
-                EventsDAO.updateEvent(event, update, callback);
+                // The DAO will make inline changes to the `event` instance. If we wish
+                // to provide the old instance in the emitted event we need to take a copy here
+                var oldEvent = _.clone(event.dataValues);
+                EventsDAO.updateEvent(event, update, function(err, updatedEvent) {
+                    if (err) {
+                        return callback(err);
+                    }
+
+                    EventsAPI.emit(EventsConstants.events.UPDATED_EVENT, ctx, oldEvent, updatedEvent);
```

Comment:
Shouldn't we als regenerate the summary when a new event is created (under a serie)?